test/unit/full_text_search/searcher_test.rb

```diff
@@ -114,5 +114,73 @@ def test_order_registered_time_asc
                        .order(created_on: :asc)
       assert_equal(ordered_news, searched_news)
     end
+
+    def test_order_last_modified_time_desc
+      parameters = {
+        order_target: "last_modified_time",
+        order_type: "desc",
+        news: "1",
+        attachments: "0",
+        limit: "-1"
+      }
+      targets = search(parameters).records
+      searched_news = targets.collect(&:source_record)
+      ordered_news = @project
+                       .news
+                       .order(created_on: :desc)
+      assert_equal(ordered_news, searched_news)
+    end
+
+    def test_order_last_modified_time_asc
+      parameters = {
+        order_target: "last_modified_time",
+        order_type: "asc",
+        news: "1",
+        attachments: "0",
+        limit: "-1"
+      }
+      targets = search(parameters).records
+      searched_news = targets.collect(&:source_record)
+      ordered_news = @project
+                       .news
+                       .order(created_on: :asc)
+      assert_equal(ordered_news, searched_news)
+    end
+
+    def test_order_score_desc
+      Issue.destroy_all
+      issue_with_high_score = Issue.generate!(description: "score score score")
+      issue_with_middle_score = Issue.generate!(description: "score score")
+      issue_with_low_score = Issue.generate!(description: "score")
+      parameters = {
+        q: "score",
```

Comment:
Could you use a different word?
`"score"` is the same value as `order_target`. It confuses us.

Reply:
fix: [9180540](https://github.com/clear-code/redmine_full_text_search/pull/121/commits/91805406c2a79643238b26e8d3a3fc35657a06e4)